doc/source/reference/command-line.rst

```diff
@@ -208,29 +197,50 @@ command-line.
 rpdb
 ++++
 
-Connect to a currently running (on localhost) rpd session.
+Connect to a currently running (on localhost) rpd session. See
+:ref:`remote-debugging` for more info.
 
 notebook
 ++++++++
 
-Launches an IPython notebook server and prints out instructions on how to open
-an ssh tunnel to connect to the notebook server with a web browser.  This is
-most useful when you want to run an IPython notebook using CPUs on a remote
+Launches a Jupyter notebook server and prints out instructions on how to open
+an ssh tunnel to connect to the notebook server with a web browser. This is
+most useful when you want to run a Jupyter notebook using CPUs on a remote
 host.
 
 stats
 +++++
 
 This subcommand provides you with some basic statistics on a given dataset.
 It provides you with the number of grids and cells in each level, the time
-of the dataset, the resolution, and the maximum density in a variety of units.
-It is tantamount to performing the ``print_stats()`` inside of yt.
+of the dataset, and the resolution. It is tantamount to performing the
+``print_stats()`` inside of yt.
+
+Additionally, there is the option to print the minimum, maximum, or both for
+a given field. The field is assumed to be density by default:
+
+.. code-block:: bash
+
+   yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max --min
+
+or a different field can be specified using the ``-f`` flag:
+
+.. code-block:: bash
+
+   yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max --min -f temperature
+
+The field-related stats output from this command can be directed to a file using
+the ``-o`` flag:
+
+.. code-block:: bash
+
+   yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max -o out_stats.dat
```

Comment:
Does it do anything different than this ?
```
yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max > out_stats.dat
```

Reply:
Yes--it outputs only the max and min stats information. 

In my opinion this option deserves a refactor, but out of scope for this PR.

Reply:
Yeah, that's totally fine.